deploy/service_monitor.yaml

```diff
@@ -14,3 +14,34 @@ spec:
   selector:
     matchLabels:
       name: marketplace-operator
+---
```

Comment:
If these are needed for the cr, why aren't we applying them first in separate files?

Reply:
I was following the [example set in OLM](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/ocp/manifests/0.11.0/0000_90_olm_00-service-monitor.yaml#L63-L93).